lib/ts/recipe/passwordless/recipe.ts

```diff
@@ -73,41 +73,11 @@ export default class Recipe extends RecipeModule {
 
         this.emailDelivery =
             ingredients.emailDelivery === undefined
-                ? new EmailDeliveryIngredient(this.config.emailDelivery)
+                ? this.config.contactMethod === "PHONE"
+                    ? undefined
+                    : new EmailDeliveryIngredient(this.config.emailDelivery)
                 : ingredients.emailDelivery;
```

Comment:
There should always be an instance of this in the recipe. Only in typeinput (from the user's point of view), emailDelivery is optional.

If an instance of `emailDelivery` is passed in the constructor, it should always be assigned

During the normlising process, if the `contactMethod === "PHONE"`, then you can just create a backward or empty service and assign this variable.

---------------
The above is one option. The other option is that we don't store `emailDelivery` as an instance variable of the recipe class and only have it in the normalised config (depending on the `contactMethod`). But this way, it will require that you assign the constructor passed version to the normalised config's emailDelivery instance like:
```ts
if (normalisedConfig. contactMethod !== "PHONE" && ingredients.emailDelivery !== undefined) {
   normalisedConfig.emailDelivery = ingredients.emailDelivery;
}
```

I am fine with either, but I prefer the first option as it's consistent with other recipes.